specification/eventgrid/resource-manager/Microsoft.EventGrid/preview/2019-02-01-preview/EventGrid.json

```diff
@@ -2993,8 +3393,7 @@
                     "x-ms-enum": {
                         "name": "InputSchema",
                         "modelAsString": true
-                    },
-                    "default": "EventGridSchema"
+                    }
```

Comment:
Would want to retain the "default": "EventGridSchema" that we have in the previous API version.

Reply:
fixed